<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Latitude" name="com.Actouch.DirectInvoiceINV.TestScript.DirectInvoiceLocation" tests="2" failures="1" timestamp="13 Jan 2018 06:06:54 GMT" time="23.020" errors="0">
  <testcase name="DIINV_Login_TC_1" time="10.808" classname="com.Actouch.DirectInvoiceINV.TestScript.DirectInvoiceLocation"/>
  <testcase name="DIINV_MultiProduct_TC_16" time="12.212" classname="com.Actouch.DirectInvoiceINV.TestScript.DirectInvoiceLocation">
    <failure type="java.io.FileNotFoundException" message="D:\Selenium\Direct Invoice.xlsx (The system cannot find the file specified)">
      <![CDATA[java.io.FileNotFoundException: D:\Selenium\Direct Invoice.xlsx (The system cannot find the file specified)
	at java.io.FileInputStream.open(Native Method)
	at java.io.FileInputStream.<init>(FileInputStream.java:138)
	at java.io.FileInputStream.<init>(FileInputStream.java:97)
	at com.Actouch.Sales.generic_Lib.SO_Excellib.getExcelDat(SO_Excellib.java:58)
	at com.Actouch.DirectInvoiceINVLoc.pageObjectRepository.DIMultiProd.payltr(DIMultiProd.java:92)
	at com.Actouch.DirectInvoiceINV.TestScript.DirectInvoiceLocation.DIINV_MultiProduct_TC_16(DirectInvoiceLocation.java:384)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:601)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
    </failure>
  </testcase> <!-- DIINV_MultiProduct_TC_16 -->
</testsuite> <!-- com.Actouch.DirectInvoiceINV.TestScript.DirectInvoiceLocation -->
